admin/python/lsst/qserv/admin/partConfig.py

```diff
@@ -184,6 +184,8 @@ def cssTableOptions(self):
             options['subChunks'] = bool(int(self.get('part.subChunks', 1)))
             options['dirDb'] = self['dirDb']
             options['dirTable'] = self['dirTable']
+            options['lockInMem'] = 'lockInMem' in self ? self['lockInMem'] : False
+            options['scanSpeed'] = 'scanSpeed' in self ? self['scanSpeed'] : 0
```

Comment:
This looks like C++ except that `self` needs to be replaced with `this` :) This won't compile unfortunately, I'd replace it with more pythonic:

    options['lockInMem'] = self.get('lockInMem', False)
    options['scanSpeed'] = self.get('scanSpeed', 0)